src/com/flickity.js

```diff
@@ -165,10 +165,13 @@
 				}
 				
 				App.callback(o.inited);
-			} else if (slider.find(o.cellSelector.length == 1)) {
+			} else if (slider.find(o.cellSelector).length === 1) {
 				slider.addClass(o.abortedClass);
 				slider.find(o.cellSelector).addClass(o.selectedClass);
 				App.callback(o.aborted);
+			} else {
+				App.log('No flickity found or multiple flickity in the scope');
+				App.callback(o.aborted);
```

Comment:
Should we try to always add ALL values in the default object ?

Reply:
?

Reply:
aborted is not declared in `defaultOptions`. Should we define in ?

Reply:
@PascalPiche  ???

Reply:
@PascalPiche  Wanna add a commit that does it ?